packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Promo.tsx

```diff
@@ -65,11 +65,11 @@ export const WithText = (props: PromoProps): JSX.Element => {
           <div className="promo-text">
             <div>
               <div className="field-promotext">
-                <Text className="promo-text" field={props.fields.PromoText} />
+                <Text className="promo-text" field={props.fields.PromoText} encode={false} />
```

Comment:
What is the reason do disable encoding?

Reply:
I reverted and changed Text component to RichText. In PromoText we use html-text